numpy/f2py/f2py2e.py

```diff
@@ -269,8 +269,8 @@ def scaninputline(inputline):
             options["f2py_wrapper_output"] = l
         elif f == 1:
             try:
-                open(l).close()
-                files.append(l)
+                with open(l) as f:
+                    files.append(l)
```

Comment:
This was just fine as it was before, albeit strange.

Reply:
IMO, this is cleaner, less strange and preserves functionality

Reply:
Please either undo this change, or switch to
```
with open(l) as f:
    pass
files.append(l)
```